lib/project.js

```diff
@@ -53,10 +54,10 @@ function Project(pkgPath, cb) {
 function Project$gather(cb) {
   var project = this;
   if (this.git) {
-    this.git.remotes(function(err, remotes) {
+    this.git.remotes(function (err, remotes) {
```

Comment:
you are changing whitespace to non-standard, or maybe its a problem with editor settings?